Fatras/Geant4/include/ActsFatras/Geant4/DummyDetectorConstruction.hpp

```diff
@@ -0,0 +1,43 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "G4RunManager.hh"
+#include "G4VUserDetectorConstruction.hh"
+
+class G4LogicalVolume;
+class G4VPhysicalVolume;
+
+namespace ActsFatras {
+
+/// @brief Convenience method assuring the existance of a G4RunManager
+///
+/// @return Pointer to the run manager
+G4RunManager* makeDummyRunManager();
```

Comment:
```suggestion
/// Either return the global run manager or create a new dummy one.
///
/// @return Pointer to the run manager
G4RunManager* ensureGeant4RunManager();
```
I initially envisioned that this function only creates the dummy run manager. Since you do both, I would suggest to reflect that in the name.

Reply:
Done